spec/controllers/ems_cloud_controller_spec.rb

```diff
@@ -415,6 +415,13 @@
         is_expected.to render_template(:partial => "layouts/listnav/_ems_cloud")
       end
     end
+
+    it 'displays only associated storage_managers' do
+      FactoryGirl.create(:ems_storage, :name => 'abc', :type =>  "ManageIQ::Providers::Amazon::StorageManager::Ebs", :parent_ems_id => @ems.id)
+      FactoryGirl.create(:ems_storage, :name => 'xyz', :type =>  "ManageIQ::Providers::Amazon::StorageManager::Ebs", :parent_ems_id => @ems.id)
+      get :show, :params => { :display => "storage_managers", :id => @ems.id, :format => :js }
```

Comment:
I would have really liked if there was a way to test the number of rows being rendered by 
`?display=storage_managers` for accuracy. (The test that we discussed in the UI room related to #2360 )

Reply:
Please, have a shot at that. As I wrote on gitter this needs an integration test.

You create some markup, that markup leads to a component being loaded in the browser, the component fires a request and response to that request need to have certain properties. If you have cycles, I'd love to review such test and adapt it in more places. It would be a great contribution.

Reply:
All I can say is I scanned the `response` object for the things like `@pages={:current=>1, :items=>2, :total=>1}`, but no luck there. 
Nothing conclusive to rely on.

Also no free cycles at the moment - I'm completely swamped with other priorities that we need for the release.
I'm sure you will find someone else to take the initiative to write this helpful spec :)